shared/js/background/chrome-events.es6.js

```diff
@@ -56,10 +64,32 @@ chrome.webRequest.onHeadersReceived.addListener(
             // returns a promise
             return ATB.updateSetAtb(request)
         }
+
+        const activeExperiment = settings.getSetting('activeExperiment')
+        if (activeExperiment) {
+            const experiment = settings.getSetting('experimentData')
+
+            if (experiment && experiment.blockingActivated) {
+                // Strip 3rd party response header
+                const tab = tabManager.get({ tabId: request.tabId })
+                if (!request.responseHeaders) return
+                if (tab && tab.site.whitelisted) return
+                if (tab && utils.isFirstParty(request.url, tab.url)) return
+                const index = request.responseHeaders.findIndex(header => { return header.name.toLowerCase() === 'set-cookie' })
+                if (index !== -1) {
+                    if (!cookieConfig.isExcluded(request.url) && trackerutils.isTracker(request.url)) {
+                        request.responseHeaders.splice(index, 1)
```

Comment:
I found out that this will only remove singe `set-cookie` header while in the wild there can be multiple:

![3-set-cookie](https://user-images.githubusercontent.com/985504/108744761-99927f80-753a-11eb-8044-3fa761ffabaa.jpg)

We should probably `.filter()` all of those out here.